debian/homestead-prov.postinst

```diff
@@ -79,6 +81,14 @@ case "$1" in
         [ ! -x /etc/init.d/clearwater-secure-connections ] || /etc/init.d/clearwater-secure-connections reload
     ;;
 
+    triggered)
+        # Restart homestead-prov.  Always do this by terminating homestead-prov so monit will
+        # restart it more-or-less immediately.  (monit restart seems to have
+        # significant lag.)
+        # Don't fail if it's already stopped.
+        service homestead-prov stop || true
+    ;;
```

Comment:
Can you send out a PSA on triggering/activating to the Clearwater team?  We might want to use this technology for other places were we restart a service when another service is installed (e.g. installing sproutlets on top of sprout or adding monit config files).